### PR TITLE
update quickstart guide for monitoring dashboards

### PR DESCRIPTION
**Pull request by umangachapagain**
Some changes to Quickstart guide to match new design changes for release-4.8.

Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

### CODE REVIEW

**Comment by bipuladh** [comment]
Thank you @umangachapagain 
/lgtm

File: controllers/storagecluster/quickstart_constants.go [hunk 1]
```diff
@@ -111,13 +111,13 @@ spec:
       description: "You can monitor any storage resources managed by OpenShift Container Storage on the Persistent Storage and Object Service dashboards.
 
 
-   Click Home -> Overview to get to the dashboard page, then click on the appropriate tab.
+   Click Storage -> Overview to get to the dashboard page, then click on the appropriate tab.
 
 
-   The Persistent Storage dashboard tab shows the state of OpenShift Container Storage as a whole, as well as the state of any persistent volumes.
+   The Block&File dashboard tab shows the state of OpenShift Container Storage as a whole, as well as the state of any persistent volumes.
```
Comment (jarrpa) [inline controllers/storagecluster/quickstart_constants.go:117]: This looks weird, but just to verify: Is this the correct name for this dashboard, e.g. with no spaces?

Comment (umangachapagain) [reply controllers/storagecluster/quickstart_constants.go]: Apparently not. Updated to `Block and File`.

**Review by jarrpa — APPROVED** [review]
/lgtm

**Comment by jarrpa** [comment]
| /test ocs-operator-ci
-- | --

**Pull request merged**